tests/test_date.py

```diff
@@ -84,224 +126,348 @@ def test_date_arguments_and_date_range_with_default_post_days(self):
               expected_results=[datetime(2014, 12, 1)]),
     ])
     def test_should_one_date_for_each_month(self, low, high, expected_results):
```

Comment:
What about `test_day_should_be_only_one_date_representing_each_month_when_period_is_month` then?

In your variant it is not very clear what `one` should it be.